### ISSUE TITLE
Document important options

### ISSUE THREAD

**Issue by bcomnes** [yoshuawuyts/bankai#87]
Im having some issues figuring out how to port over an older bankai app to using 5.0.0 with the current set of docs.

How would I port over the following?

```js
var bankai = require('bankai')()
var browserify = require('browserify')
var staticClient = require('./client/app')
var js = bankai.js(browserify, require.resolve('./client'), {debug: true})
var html = bankai.html({ favicon: false, css: false })
```
(from https://github.com/bcomnes/hyperblog/blob/master/index.js)

README should also probably link to the full set of possible options for `css`, `js` and `html`, as well as include some of the more common options.

**Comment by yoshuawuyts** [comment]
This is what it was intended like:
```js
const bankai = require('bankai')
const assets = bankai(require.resolve('./client'), {
  html: { favicon: false },
  js: { debug: true },
  css: false
})
const js = assets.js
const html = assets.html
```

**Comment by yoshuawuyts** [comment]
@bcomnes does this resolve it for you?

**Comment by bcomnes** [comment]
The example here does yeah.  I still think the read me or some kind of example should include this info though so its discoverable.  I can PR some suggestions when I have time to work on projects using the lib again.

**Comment by bcomnes** [comment]
Ok this is a weird one:

```js
const bankai = require('bankai')

const assets = bankai(require.resolve('./client'), {
  html: { favicon: false },
  js: { debug: true },
  css: false
})


const js = assets.js
const html = assets.html

http.createServer((req, res) => {
  switch (req.url) {
    case '/': return html(req, res).pipe(res)
    case '/bundle.js': return js(req, res).pipe(res)
    default: return (res.statusCode = 404 && res.end('404 not found'))
  }
}).listen(8080)
```

Does not work:

```
TypeError: this._createJs is not a function
   at Bankai.js (/Users/bret/repos/hyperblog/node_modules/bankai/index.js:44:8)
   at RouteResult.handler (/Users/bret/repos/hyperblog/index.js:46:8)
   at Hyperserv.handleRequest [as router] (/Users/bret/repos/hyperblog/node_modules/@bret/http-hash-router/index.js:68:22)
   at routeReq (/Users/bret/repos/hyperblog/node_modules/hyperserv/index.js:37:14)
   at Hyperserv.logger (/Users/bret/repos/hyperblog/node_modules/morgan/index.js:144:5)
   at Server.onReq (/Users/bret/repos/hyperblog/node_modules/hyperserv/index.js:32:10)
   at emitTwo (events.js:106:13)
   at Server.emit (events.js:191:7)
   at HTTPParser.parserOnIncoming [as onIncoming] (_http_server.js:547:12)
   at HTTPParser.parserOnHeadersComplete (_http_common.js:99:23)
```

But this does:

```js
const bankai = require('bankai')

const assets = bankai(require.resolve('./client'), {
  html: { favicon: false },
  js: { debug: true },
  css: false
})

http.createServer((req, res) => {
  switch (req.url) {
    case '/': return assets.html(req, res).pipe(res)
    case '/bundle.js': return assets.js(req, res).pipe(res)
    default: return (res.statusCode = 404 && res.end('404 not found'))
  }
}).listen(8080)
```

does work! 

Scratching my head at where I might have missed something obvious.  Any ideas?  Here is the repo I'm screwing around on:

https://github.com/bcomnes/hyperblog/tree/greenkeeper-bankai-5.0.0

**Comment by yoshuawuyts** [comment]
Landed `v5.1.0` - could you give this another shot?

**Comment by yoshuawuyts** [comment]
gonna assume this is no longer an issue (:
